### DISCUSSION TITLE
Copilot Access Issue

### DISCUSSION THREAD

**Discussion by IamNaser** [community/community#109400] [GitHub Education]
### Select Topic Area

Question

### Body

Hello everyone,

I am an Instructor. I am motivating and encouraging my students in using Github and explore the features of github copilot.

But I am not receiving my access to github copilot as an Instructor.
Please help me in this regard.

**Comment by paulvickers** [comment]
Same here.

**Reply by avsmith** [reply]
Same here. Has there been any feedback from Github?
